src/Redux/todo-reducer.ts

```diff
@@ -5,68 +5,72 @@ export type TodoActionsType = InferActionTypes<typeof todoActions>
 export type TodoListItemType = {
     id: number,
     task: string,
-    progress: boolean
+    progress: boolean,
 }
 
 export type TodoInitialStateType = {
-    todoList: TodoListItemType[]
-    progressAll: boolean 
-    changedFilter: string
+    todoList: TodoListItemType[],
+    progressAll: boolean,
+    changedFilter: string,
+    namesFilters: string[],
 }
 
 const initialState: TodoInitialStateType = {
     todoList: [],
     progressAll: false,
     changedFilter: 'all',
+    namesFilters: ['all', 'active', 'completed'],
+}
+
+const getIndexById = (id: number, array: TodoListItemType[]) => {
+    const index = array.findIndex(element => {
+        return element.id === id
+    })
+    return index
+}
```

Comment:
This is too complicated. You are creating a wrapper for a function findIndex. It's not needed.